dashboard/src/components/Catalog/CatalogItems.tsx

```diff
@@ -14,13 +14,20 @@ export default function CatalogItems({ charts, csvs, cluster, namespace }: ICata
   const chartItems: ICatalogItemProps[] = useMemo(
     () =>
       charts.map(c => {
+        let icon;
+        if (c.attributes?.icon) {
+          icon = c.attributes.icon.startsWith("/")
+            ? c.attributes.icon
+            : "/".concat(c.attributes.icon);
+          icon = "api/assetsvc".concat(icon);
```

Comment:
perhaps the above 4 lines could be just:
```
icon = "api/assetsvc/".concat(icon.replace(/^\//, "");
```

Reply:
or more simply: `api/assetsvc/${trimStart(c.attributes.icon, "/")}`

Again `trimStart` from lodash :) whenever you need to do something with strings, arrays or objects, lodash have many useful functions.